ark-grpc/src/client.rs

```diff
@@ -63,6 +65,9 @@ pub struct Client {
     url: String,
     ark_client: Option<ArkServiceClient<tonic::transport::Channel>>,
     indexer_client: Option<IndexerServiceClient<tonic::transport::Channel>>,
+    admin_client: Option<
+        generated::ark::v1::admin_service_client::AdminServiceClient<tonic::transport::Channel>,
+    >,
```

Comment:
Wait, what? Do we need the admin client to create notes?? Surely the admin client is for whoever is controlling `arkd`.

We used to have the admin client dependency, but it was never used and it doesn't seem to make sense for this repo, so we removed it.